extensions/Worker.Extensions.Storage.Blobs/src/BlobStorageConverter.cs

```diff
@@ -15,12 +15,40 @@
 using Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs;
 using Microsoft.Extensions.Options;
 using Microsoft.Extensions.Logging;
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
 
 namespace Microsoft.Azure.Functions.Worker
 {
     /// <summary>
     /// Converter to bind Blob Storage type parameters.
     /// </summary>
+    [SupportsDeferredBinding]
+    [SupportsJsonDeserialization]
+    [SupportedConverterType(typeof(string))]
+    [SupportedConverterType(typeof(string[]))]
+    [SupportedConverterType(typeof(IEnumerable<string>))]
+    [SupportedConverterType(typeof(Stream))]
+    [SupportedConverterType(typeof(Stream[]))]
+    [SupportedConverterType(typeof(IEnumerable<Stream>))]
+    [SupportedConverterType(typeof(byte[]))]
+    [SupportedConverterType(typeof(byte[][]))]
+    [SupportedConverterType(typeof(IEnumerable<byte[]>))]
+    [SupportedConverterType(typeof(BlobBaseClient))]
+    [SupportedConverterType(typeof(BlobBaseClient[]))]
+    [SupportedConverterType(typeof(IEnumerable<BlobBaseClient>))]
+    [SupportedConverterType(typeof(BlobClient))]
+    [SupportedConverterType(typeof(BlobClient[]))]
+    [SupportedConverterType(typeof(IEnumerable<BlobClient>))]
+    [SupportedConverterType(typeof(BlockBlobClient))]
+    [SupportedConverterType(typeof(BlockBlobClient[]))]
+    [SupportedConverterType(typeof(IEnumerable<BlockBlobClient>))]
+    [SupportedConverterType(typeof(PageBlobClient))]
+    [SupportedConverterType(typeof(PageBlobClient[]))]
+    [SupportedConverterType(typeof(IEnumerable<PageBlobClient>))]
+    [SupportedConverterType(typeof(AppendBlobClient))]
+    [SupportedConverterType(typeof(AppendBlobClient[]))]
+    [SupportedConverterType(typeof(IEnumerable<AppendBlobClient>))]
+    [SupportedConverterType(typeof(BlobContainerClient))]
```

Comment:
We should make a new GH issue to break this converter out into one converter per type

Reply:
Done - https://github.com/Azure/azure-functions-dotnet-worker/issues/1512